qabel_register/settings.py

```diff
@@ -125,3 +125,7 @@
 
 # Pending update requests expire after this time interval
 PENDING_REQUEST_MAX_AGE = datetime.timedelta(days=3)
+
+# This is the key the server uses to receive signed requests. By default it's set to a random key at each startup,
```

Comment:
Yep. Through this I even understand now exactly why that works with the ECDH's "cross-over" symmetry (shared = dh(my.private, other.public) = dh(other.private, my.public)).